README.md

```diff
@@ -25,6 +25,8 @@ marking unsafe functions with a `*`, suppose your function `f` calls functions
 Functions with high badness have a lot of opportunities to be memory unsafe.
 
 ## Installation
+(More detailed instructions for installing on Ubuntu can be found at [UbuntuInstallation.md](./docs/UbuntuInstallation.md))
+
```

Comment:
Nitpick: maybe move this below the other installation instructions, since it's more specific than everything else?

Reply:
Thank you I have moved the position to the bottom of the section